tests/functional/labels-auth.js

```diff
@@ -109,6 +109,35 @@ registerSuite("Labels (auth)", {
           assert.notInclude("is-active", className);
         })
         .end();
+    },
+
+    "Label editor filters label list based on entered text": function() {
+      var count = 0;
+      return FunctionalHelpers.openPage(
+        this,
+        url("/issues/100"),
+        ".js-LabelEditorLauncher",
+        true /* longerTimeout */
+      )
+        .findByCssSelector(".js-LabelEditorLauncher")
+        .click()
+        .end()
+        .findByCssSelector(".js-label-search")
+        .type("tacos")
+        .end()
+        .sleep(1000)
+        .findAllByXpath(
```

Comment:
https://theintern.io/docs.html#Leadfoot/2/api/Command/finddisplayedbycssselector is probably another way to only grab visible docs, but this is working. So...

Reply:
The problem with `findDisplayedByCssSelector` (iirc) is that it will only grab the first element that qualifies, so it could still be displaying others. I tried using `findAllByCssSelector`, then `isDisplayed`, but it wasn't working for some reason (all were displayed). I did eventually put in a short sleep after I switched tactics, though, so it might have worked with that delay.